Library/DensityRidgeline.Rmd

```diff
@@ -9,6 +9,9 @@ knitr::opts_chunk$set(echo = TRUE)
 ```
 
 ```{r DensityRidgeline, fig.path = '../docs/figure/', warning = FALSE}
+# Objective ---------------
+  # Create a group of Ridgeline density plots comparing distributions of health expenditures by timing and treatment status. 
+
```

Comment:
Add proper citation of the book in here: https://openknowledge.worldbank.org/handle/10986/25030

Reply:
Original data can be downloaded at http://pubdocs.worldbank.org/en/984951475693055950/HISP.zip